src/modules/config.js

```diff
@@ -26,4 +51,7 @@ export const
     supportedAudio = config.supportedAudio,
     celebrations = config.celebrations,
     links = config.links,
-    sponsors = config.sponsors
+    sponsors = config.sponsors,
+    mode = (process.env.API_URL && !process.env.WEB_URL) ? 'API' :
+        (process.env.WEB_URL && process.env.API_URL) ? 'WEB' : undefined,
```

Comment:
you could use `apiURL` and `webURL` from above here

Reply:
yep, forgot about them, will do